### ISSUE TITLE
Allow configuration of timeouts when creating go pool

### ISSUE THREAD

**Issue by alberto-andria** [go-goracle/goracle#183]
We have a critical app that requires "high availability" so in order to support a DB failure, we have configured an oracle RAC. The failover strategy is described in the connection string: 
```
(DESCRIPTION_LIST=(LOAD_BALANCE=off)(FAILOVER=on)
    (DESCRIPTION=(ADDRESS_LIST=(LOAD_BALANCE=on)(ADDRESS=(PROTOCOL=TCP)(HOST=exampleDB1)(PORT=1521)))(CONNECT_DATA=(SERVICE_NAME=exampleDB1_SN)))
    (DESCRIPTION=(ADDRESS_LIST=(LOAD_BALANCE=on)(ADDRESS=(PROTOCOL=TCP)(HOST=exampleDB2)(PORT=1521)))(CONNECT_DATA=(SERVICE_NAME=exampleDB2_SN)))
    (DESCRIPTION=(ADDRESS_LIST=(LOAD_BALANCE=on)(ADDRESS=(PROTOCOL=TCP)(HOST=exampleDB3)(PORT=1521)))(CONNECT_DATA=(SERVICE_NAME=exampleDB3_SN))))
```

But in order to get this to work properly we need to change the timeouts in the file drv.go:
```go
poolCreateParams.timeout = 300             // seconds before idle pool sessions got evicted
poolCreateParams.waitTimeout = 3 * 1000    // milliseconds to wait for a session become available
poolCreateParams.maxLifetimeSession = 3600 // maximum time in seconds till a pooled session may exist
```
We consider that the solution will be to pass these timeouts using the configuration string adding three more parameters.

For example 
```
//   ora://login:password@sid/? \
//	   timeout=300& \
//	   waitTimeout=3000& \
//	   maxLifetimeSession=3600
```

If you agree on this approach, we will more than happy to make a pull request with the changes

Greetings from Argentina :)

**Comment by tgulacsi** [comment]
Sure, why not.
Just give them names that points where they're used: poolSessionTimeout, poolWaitTimeout, pollSessionMaxLifetime.
And apply sane defaults.

**Comment by tgulacsi** [comment]
Thanks to @alberto-andria it is now implemented in v2.18.0 (df8cb6f6369f5ac04dbf15ef19b18cb448abb27e).
